packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -490,6 +508,15 @@ describe('Base Deploy Retrieve Commands', () => {
       expect(executor.retrieveStub.callCount).to.equal(1);
     });
 
+    it('should call setup cancellation logic', async () => {
```

Comment:
I wanted to verify that a cancel command would call the .cancel method on the operation. However I wasn't able to figure out how to do this within the baseDeployRetrieve tests.